src/common/containers/landing.js

```diff
@@ -27,7 +27,7 @@ class Landing extends Component {
 
               <ul className={ styles.banner }>
                 <li className={ styles.bannerImage }>
-                  <img src='https://assets.ubuntu.com/v1/ed6d1c5b-build.snapcraft.hero.svg' alt="Snapcraft hero image, push to GitHub, built automatically, release for your users."/>
+                  <img src='https://assets.ubuntu.com/v1/ed6d1c5b-build.snapcraft.hero.svg' alt=""/>
```

Comment:
Again, alt text should be used for when the image can't be displayed or users that are using a screen reader and are vision impared. Removing the alt text is a detriment for this, instead of removing the alt can you look to improve it.

A good example post of how to write alt attribute content can be found here https://www.phase2technology.com/blog/no-more-excuses-definitive-guide-alt-text-field